File: testdata/GenericParameter.java
package x.y.z;

import java.util.List;
import java.util.Map;

public class GenericParameter
{
  public void setAddresses
  (
    List<Address> addresses
  )
  {
    this.addresses = addresses
  }


  protected void
  setAddressMap
  (
    Map<String, Address> addressMap
  ) 
  {
  }

  private List<Address> addresses = null;
}

File: testdata/expected/valswithtypes/GenericParameter.scala
package x.y.z

import java.util.List
import java.util.Map

class GenericParameter
{
  def
  setAddresses
  (
    addresses : List[Address]
  ) : Unit =
  {
    this.addresses = addresses
  }


  protected def
  setAddressMap
  (
    addressMap : Map[String, Address]
  ) : Unit =
  {
  }

  private var addresses : List[Address] = null
}
